action.yml

```
@@ -20,6 +20,8 @@ inputs:
 outputs:
   tf_actions_plan_has_changes:
     description: 'Whether or not the Terraform plan contained changes.'
+  tf_outputs:
```

Following the suggested prefix. Also making singular.

```suggestion
  tf_actions_output:
```